cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/dataset/BusinessMetadataDataset.java

```diff
@@ -147,4 +241,11 @@ private String getTargetType(Id.NamespacedId namespacedId) {
     }
     return namespacedId.getClass().getSimpleName();
   }
+
+  private String getMetadataKey(byte [] rowKey) {
+    MDSKey.Splitter keySplitter = new MDSKey(rowKey).split();
+    keySplitter.skipString();
```

Comment:
It would be good to have a visual representation of the key components in comments.